central/networkgraph/service/service_impl.go

```diff
@@ -429,7 +429,9 @@ func (s *serviceImpl) addDeploymentFlowsToGraph(
 		}
 	}
 
-	flows, _, err := flowStore.GetMatchingFlows(networkGraphGenElevatedCtx, pred, request.GetSince())
+	since := protocompat.ConvertTimestampToTimeOrNil(request.GetSince())
```

Comment:
(nit) is the introduction of the `since` var necessary? I think, we may remove it.